lib/src/Uncertainty/Process/ARMAFactoryImplementation.cxx

```diff
@@ -155,24 +155,12 @@ ARMA ARMAFactoryImplementation::build(const TimeSeries & timeSeries) const
   throw NotYetImplementedException(HERE) << "In ARMAFactoryImplementation::build(const TimeSeries & timeSeries) const";
 }
 
-ARMA ARMAFactoryImplementation::build(const TimeSeries & timeSeries,
-                                      Point & informationCriteria) const
-{
-  throw NotYetImplementedException(HERE) << "In ARMAFactoryImplementation::build(const TimeSeries & timeSeries, Point & informationCriteria) const";
-}
-
 /* Build method */
 ARMA ARMAFactoryImplementation::build(const ProcessSample & sample) const
 {
```

Comment:
I think we should keep only ARMAFactoryImplementation::build methods : 

  *ARMAFactoryImplementation::build(const TimeSeries & timeSeries,  Point & informationCriteria) const*

  *ARMAFactoryImplementation::build(const ProcessSample & sample,  Point & informationCriteria) const*

Both **ARMAFactoryImplementation::build(const TimeSeries & timeSeries) const** and 
  **ARMAFactoryImplementation::build(const ProcessSample & sample) const** should be removed

What do you think ?